src/components/Header/Header.module.css

```diff
@@ -0,0 +1,17 @@
+.loadFactoryHeader {
```

Comment:
We've agreed to name css modules under a component folder as `index.module.css`. Please rename this module.

Reply:
Good point. I have fixed.